develop.md

```diff
@@ -57,3 +57,15 @@ docker buildx build -t ghcr.io/nextcloud-releases/all-in-one:latest --build-arg
 ```
 1. Start a container with above built image.
 1. Since the hash of a locally built image doesn't match the latest release mastercontainer, it prompts for a mandatory update. To temporarily bypass the update suffix `?bypass_mastercontainer_update` to the URL. Eg: `https://localhost:8080/containers?bypass_mastercontainer_update`
+
+## How to locally build and test changes to other containers using the no_pull_container_ids param
+1. Push changes to your own git fork and branch
+1. Use below commands to build the container image for a custom git url and branch:
+```
+cd Containers/{container}
+docker buildx build -t ghcr.io/nextcloud-releases/all-in-one:latest --build-arg AIO_GIT_URL="https://github.com/my-fork-repo/all-in-one.git" --build-arg AIO_GIT_BRANCH="my-feature-branch" --load .
+```
+1. Stop the containers using the AIO admin interface
+1. Since the hash of a locally built image doesn't match the latest release. Reload the AIO admin interface with the param `no_pull_container_ids={container_id}`, e.g. `?no_pull_container_ids=nextcloud-aio-apache,nextcloud-aio-nextcloud`.
+  - `container_id` matches the `container_name` from `containers.json`
```

Comment:
this docs need to be adjusted then as well. For example it should be `docker buildx build -t ghcr.io/nextcloud-releases/{container}:latest --build-arg AIO_GIT_URL="https://github.com/my-fork-repo/all-in-one.git" --build-arg AIO_GIT_BRANCH="my-feature-branch" --load .`, no?